src/TouristInfoCards.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+function TouristInfoCards() {
+  return (
+    <div className="cards-wrapper">
+      <div className="card info-card">
+        <img
+          src="https://images.unsplash.com/photo-1546867042-cdba19743124?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MjkxfHxnbGFzZ293fGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60"
+          className="card-img-top"
+          alt="Glasgow"
```

Comment:
If you want to make this code shorter and easier to maintain, you could use a `map` instead of writing the same HTML structure three times.

```js
const cityInfo = [
    {name: "Glasgow", text: "Glasgow is the largest city in Scotland and ...", moreInfoLink: "http://www.peoplemakeglasgow.com", imgSrc: "https://images.unsplash.com/photo-1546867042"},
    {name: "Manchester", text: "Manchester is one of the ...", moreInfoLink: "http://www.visitmanchester.com", imgSrc: "https://images.unsplash.com/photo-1546867042"},
    // ...
];
```

```js
return (
    <div className="cards-wrapper">
        { cityInfo.map(city => /* City HTML here */)  }
    </div>
)
```